src/components/DropdownMenu/component.js

```diff
@@ -0,0 +1,45 @@
+import { h, Fragment } from "preact";
```

Comment:
It's taking shape, love it!

- List items are not the same height as the dropdown button contrary to figma designs
- I think we can change the stylekit border color to match v4 (#DFE1E4)

Something I don't quite understand yet is why we need ListBox when we have DropdownMenu? Is there not significant overlap between the two? Could we merge ListBox into DropdownMenu?

Reply:
Initially, I created the `DropdownMenu` component thinking it was the best fit for the `select` element. After using it for sometime, I realized that the `ListBox` component has a better fit for the `select` element instead. Since I already started the `DropdownMenu`, I decided to keep it for future use.

The main differences between these components are:

- The `DropdownMenu` has an Icon for the button. It also has an Icon and a callback for each option. And each option can have a sub-menu.
- The `ListBox` component doesn't have icons or callbacks per option. It is pretty similar to the `select` element.

> List items are not the same height as the dropdown button contrary to figma designs
I think we can change the stylekit border color to match v4 (#DFE1E4)

Will fix the above points in a bit!